`test_create_user_and_login/test.py::test_login_as_dropped_user_xml` is flaky

Example failure (from [here](https://s3.amazonaws.com/clickhouse-test-reports/json.html?REF=25.6&sha=db3bb9bbd3dc2a26f4a43b9e7c962e84968a60c4&name_0=ReleaseBranchCI&name_1=Integration%20tests%20%28tsan%2C%203%2F6%29)):

```

>               raise Exception(
                    f"Expected either output '1' or one of errors '{expected_errors}', got output={out} and error={err}"
                )
E               Exception: Expected either output '1' or one of errors '['no user with such name', 'not found in user directories', 'User has been dropped']', got output= and error=Received exception from server (version 25.6.1):
E               Code: 492. DB::Exception: Received from 172.16.5.2:9000. DB::Exception: ID(af870242-d279-0cf2-4e91-4f60199c0a85) not found in `user directories`. Stack trace:
E               
E               0. ./contrib/llvm-project/libcxx/include/__exception/exception.h:113: Poco::Exception::Exception(String const&, int) @ 0x0000000025bffca0
E               1. ./ci/tmp/build/./src/Common/Exception.cpp:115: DB::Exception::Exception(DB::Exception::MessageMasked&&, int, bool) @ 0x0000000011a62cf4
E               2. DB::Exception::Exception(PreformattedMessage&&, int) @ 0x00000000089ec1e0
E               3. DB::Exception::Exception<String, String>(int, FormatStringHelperImpl<std::type_identity<String>::type, std::type_identity<String>::type>, String&&, String&&) @ 0x00000000089ebb66
E               4. ./ci/tmp/build/./src/Access/IAccessStorage.cpp:753: DB::IAccessStorage::throwNotFound(StrongTypedef<wide::integer<128ul, unsigned int>, DB::UUIDTag> const&, String const&) @ 0x0000000017ed22a4
E               5. ./ci/tmp/build/./src/Access/MultipleAccessStorage.cpp:295: DB::MultipleAccessStorage::readImpl(StrongTypedef<wide::integer<128ul, unsigned int>, DB::UUIDTag> const&, bool) const @ 0x0000000017f1c258
E               6. ./src/Access/IAccessStorage.h:288: std::shared_ptr<DB::User const> DB::IAccessStorage::read<DB::User>(StrongTypedef<wide::integer<128ul, unsigned int>, DB::UUIDTag> const&, bool) const @ 0x0000000014f292db
E               7. ./ci/tmp/build/./src/Interpreters/Context.cpp:1605: DB::Context::setUser(StrongTypedef<wide::integer<128ul, unsigned int>, DB::UUIDTag> const&, std::vector<StrongTypedef<wide::integer<128ul, unsigned int>, DB::UUIDTag>, std::allocator<StrongTypedef<wide::integer<128ul, unsigned int>, DB::UUIDTag>>> const&) @ 0x00000000199789f1
E               8. ./ci/tmp/build/./src/Interpreters/Session.cpp:707: DB::Session::makeQueryContextImpl(DB::ClientInfo const*, DB::ClientInfo*) const @ 0x0000000019dd6ea9
E               9. ./ci/tmp/build/./src/Interpreters/Session.cpp:646: DB::Session::makeQueryContext(DB::ClientInfo const&) const @ 0x0000000019dd6423
E               10. ./ci/tmp/build/./src/Server/TCPHandler.cpp:2189: DB::TCPHandler::processQuery(std::optional<DB::QueryState>&) @ 0x000000001cff0cfb
E               11. ./ci/tmp/build/./src/Server/TCPHandler.cpp:975: DB::TCPHandler::receivePacketsExpectQuery(std::optional<DB::QueryState>&) @ 0x000000001cfe7155
E               12. ./ci/tmp/build/./src/Server/TCPHandler.cpp:522: DB::TCPHandler::runImpl() @ 0x000000001cfd738e
E               13. ./ci/tmp/build/./src/Server/TCPHandler.cpp:2727: DB::TCPHandler::run() @ 0x000000001cfff888
E               14. ./ci/tmp/build/./base/poco/Net/src/TCPServerConnection.cpp:40: Poco::Net::TCPServerConnection::start() @ 0x0000000025d0f963
E               15. ./ci/tmp/build/./base/poco/Net/src/TCPServerDispatcher.cpp:115: Poco::Net::TCPServerDispatcher::run() @ 0x0000000025d101d2
E               16. ./ci/tmp/build/./base/poco/Foundation/src/ThreadPool.cpp:205: Poco::PooledThread::run() @ 0x0000000025c85a23
E               17. ./ci/tmp/build/./base/poco/Foundation/src/Thread.cpp:45: Poco::(anonymous namespace)::RunnableHolder::run() @ 0x0000000025c83dd0
E               18. ./base/poco/Foundation/src/Thread_POSIX.cpp:335: Poco::ThreadImpl::runnableEntry(void*) @ 0x0000000025c8218a
E               19. __tsan_thread_start_func @ 0x000000000895f428
E               20. ? @ 0x0000000000094ac3
E               21. ? @ 0x0000000000126850
E               . (ACCESS_ENTITY_NOT_FOUND)
E               (query: SELECT 1)
```

[Test case history](https://play.clickhouse.com/play?user=play#U0VMRUNUIHB1bGxfcmVxdWVzdF9udW1iZXIgYXMgcHIsIGNoZWNrX3N0YXJ0X3RpbWUsIGNoZWNrX25hbWUsIHRlc3RfbmFtZSwgdGVzdF9zdGF0dXMsIHJlcG9ydF91cmwtLSwgdGVzdF9jb250ZXh0X3JhdwpGUk9NIGNoZWNrcwpXSEVSRSAxCiAgICAtLUFORCBwdWxsX3JlcXVlc3RfbnVtYmVyID0gMAogICAgQU5EIHRlc3Rfc3RhdHVzIElOICgnRkFJTCcsICdFUlJPUicsICdCUk9LRU4nKQogICAgQU5EIHRvRGF0ZShjaGVja19zdGFydF90aW1lKSA+ICcyMDI1LTA2LTAxJwogICAgQU5EIHRlc3RfbmFtZSBsaWtlICcldGVzdF9jcmVhdGVfdXNlcl9hbmRfbG9naW4lJwogICAgLS0gQU5EIHRlc3RfY29udGV4dF9yYXcgbGlrZSAnJUFMVEVSX01FVEFEQVRBJScKT1JERVIgQlkgY2hlY2tfc3RhcnRfdGltZSBERVNDCgo=)

--- a/tests/integration/test_create_user_and_login/test.py
+++ b/tests/integration/test_create_user_and_login/test.py
@@ -81,6 +81,7 @@ EOF""",
         instance.exec_in_container(
             ["bash", "-c", "rm /etc/clickhouse-server/users.d/user_c.xml"]
         )
+        instance.query("SYSTEM RELOAD CONFIG")
 
         expected_errors = [
             "no user with such name",

